File: testdata/jennies/builders/constructor_argument/PHPBuilder/src/Sandbox/SomeStructBuilder.php
<?php

namespace Grafana\Foundation\Sandbox;

/**
 * @implements \Grafana\Foundation\Cog\Builder<\Grafana\Foundation\Sandbox\SomeStruct>
 */
class SomeStructBuilder implements \Grafana\Foundation\Cog\Builder
{
    protected \Grafana\Foundation\Sandbox\SomeStruct $internal;

    public function __construct(string $title)
    {
    	$this->internal = new \Grafana\Foundation\Sandbox\SomeStruct();
    $this->internal->title = $title;
    }

    /**
     * Builds the object.
     * @return \Grafana\Foundation\Sandbox\SomeStruct
     */
    public function build()
    {
        return $this->internal;
    }

    public function title(string $title): static
    {
        $this->internal->title = $title;
    
        return $this;
    }

}

File: testdata/jennies/builders/constructor_argument/CSharpBuilder/src/Grafana/Foundation/Sandbox/SomeStructBuilder.cs
namespace Grafana.Foundation.Sandbox;


public class SomeStructBuilder : Cog.IBuilder<SomeStruct>
{
    protected readonly SomeStruct @internal;

    public SomeStructBuilder(string title)
    {
        this.@internal = new SomeStruct();
        this.@internal.Title = title;
    }

    public SomeStructBuilder Title(string title)
    {
        this.@internal.Title = title;
        return this;
    }

    public SomeStruct Build()
    {
        return this.@internal;
    }
}
